cmd/podman/varlink/io.projectatomic.podman.varlink

```diff
@@ -53,32 +53,71 @@ type ImageSearch (
     star_count: int
 )
 
+# ListContainer is the returned struct for an individual container
+type ListContainer (
+    id: string,
+    image: string,
+    imageid: string,
+    command: []string,
+    createdat: string,
+    runningfor: string,
+    status: string,
+    ports: [][string]string,
+    rootfssize: int,
+    rwsize: int,
+    names: string,
+    labels: [string]string,
+    mounts: [][string]string,
+    containerrunning: bool,
+    namespaces: [string]string
+)
+
+# ContainerStats is the return struct for the stats of a container
+type ContainerStats (
+    container_id: string,
+    name: string,
+    cpu: float,
+    cpu_nano: int,
+    system_nano: int,
+    mem_usage: int,
+    mem_limit: int,
+    mem_perc: float,
+    net_input: int,
+    net_output: int,
+    block_output: int,
+    block_input: int,
+    pids: int
+)
+
 # System
 method Ping() -> (ping: StringResponse)
 method GetVersion() -> (version: Version)
 
 # Containers
```

Comment:
Do we have an API target to get a single container by ID?

Reply:
per our IRC conversation, I added one